iec61850serversimulator/src/test/java/com/cgi/iec61850serversimulator/SwitchingMomentCalculatorTest.java

```diff
@@ -16,275 +16,242 @@
-
-		LocalDateTime expectedTime = LocalDateTime.now().truncatedTo(ChronoUnit.MINUTES).withHour(timeOn.getHour())
-				.withMinute(timeOn.getMinute());
-		assertEquals(expectedTime, actualSwitchingMomentOn.getTriggerTime());
-		assertTrue(actualSwitchingMomentOn.isTriggerAction());
-
-		LocalDateTime expectedAfterTime = LocalDateTime.now().truncatedTo(ChronoUnit.MINUTES)
-				.withHour(timeOff.getHour()).withMinute(timeOff.getMinute());
-		assertEquals(expectedAfterTime, actualSwitchingMomentOff.getTriggerTime());
-		assertFalse(actualSwitchingMomentOff.isTriggerAction());
-
-		LocalDateTime expectedNextTime = LocalDateTime.now().plusDays(1).truncatedTo(ChronoUnit.MINUTES)
-				.withHour(timeOn.getHour()).withMinute(timeOn.getMinute());
-		assertEquals(expectedNextTime, actualSwitchingMomentNextDayOn.getTriggerTime());
-		assertTrue(actualSwitchingMomentOn.isTriggerAction());
-
-		LocalDateTime expectedNextAfterTime = LocalDateTime.now().plusDays(1).truncatedTo(ChronoUnit.MINUTES)
-				.withHour(timeOff.getHour()).withMinute(timeOff.getMinute());
-		assertEquals(expectedNextAfterTime, actualSwitchingMomentNextDayOff.getTriggerTime());
-		assertFalse(actualSwitchingMomentOff.isTriggerAction());
-
-	}
-
-	public Relay getMockRelay(int relayNr, Schedule customSchedule) {
-		Relay relay = new Relay(relayNr);
-
-		Schedule[] schedules = new Schedule[1];
-
-		// Making mock fixed time schedule.
-
-		if (customSchedule == null) {
-			// Default mock schedule:
-			// On time: 12 PM
-			// Off time: 1 PM
-			// Burning Minutes: 30
-			Schedule schedule = this.getMockLunchTimeSchedule(1);
-			schedules[0] = schedule;
-		}
-
-		else {
-			schedules[0] = customSchedule;
-		}
-
-		relay.setSchedules(schedules);
-
-		return relay;
-	}
-
-	public Schedule getMockLunchTimeSchedule(int scheduleNr) {
-
-		Schedule schedule = new Schedule(scheduleNr);
-
-		int relayNr = 1;
-		// Trigger day(s), see Enum in GXF LF Energy documentation
-		// 0 = Every day
-		int dayInt = 0;
-		// General time type for both on and off times
-		// 0 = fixed time, 1 = light sensor, 2 = astronomical time
-		int fixedTimeInt = 0;
-		// Optional: Specific time type for on time
-		// -1 = using general time type
-		int fixedTimeOn = -1;
-		// Optional: Specific time type for off time
-		// -1 = using general time type
-		int fixedTimeOff = -1;
-		LocalTime timeOn = LocalTime.of(12, 00);
-		LocalTime timeOff = LocalTime.of(13, 00);
-		int burningMinutes = 30;
-
-		schedule.setRelayNr(relayNr);
-		schedule.setDayInt(dayInt);
-		schedule.setTimeOn(timeOn);
-		schedule.setTimeOff(timeOff);
-		if (fixedTimeOn < 0 || fixedTimeOff < 0) {
-			schedule.setTimeOnTypeInt(fixedTimeInt);
-			schedule.setTimeOffTypeInt(fixedTimeInt);
-		} else {
-			schedule.setTimeOnTypeInt(fixedTimeOn);
-			schedule.setTimeOffTypeInt(fixedTimeOff);
-		}
-		schedule.setBurningMinsOn(burningMinutes);
-		schedule.setEnabled(true);
-
-		return schedule;
-	}
+    /**
```

Comment:
Put the JavaDoc just about the "public class ...." line.

Reply:
Done.